src/Header/OperateData.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_SRC_OPERATEDATA_H_
+#define GROUP4_SRC_OPERATEDATA_H_
+
+#include <string>
+#include <unordered_map>
+
+// This class is a self-defined data structure containing header and body.
+// User should choose to use only as request data or response data
+// OperateData newData = OperateData();
+// OperateData["key"] = "value";
+class OperateData {
+ public:
+  OperateData();
+  OperateData(OperateData &right);
+  std::unordered_map<std::string, std::string> header;
```

Comment:
Please add comment or variable on where data other than header or body is.
i.e. HTTP Request Method, Request URL, and Response Status Code.

Reply:
done.